saas/mic-hex-escs/domain/src/test/java/com/sokima/saas/mic/hex/escs/domain/transformer/clean/Tuple2CleanerResponseTransformerTest.java

```diff
@@ -0,0 +1,41 @@
+package com.sokima.saas.mic.hex.escs.domain.transformer.clean;
+
+import com.sokima.lib.building.block.transformer.MergingTransformer;
+import com.sokima.lib.ecommerce.java.domain.model.Cart;
+import com.sokima.lib.ecommerce.java.domain.model.record.CartRecord;
+import com.sokima.saas.mic.hex.escs.domain.payload.clean.CleanCartPayload;
+
+import java.time.Instant;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class Tuple2CleanerResponseTransformerTest {
+
+    Tuple2CleanerResponseTransformer tuple2CleanerResponse;
+
+    @BeforeEach
+    void setUp() {
+
+        this.tuple2CleanerResponse = new Tuple2CleanerResponseTransformer();
+    }
+
+    @Test
+    void transform_mappingValid_True() {
```

Comment:
missing case for unhappy path. 

e.g. 
original [1, 2, 3, 3, 3]

payload = [1, 2, 3, 3]
happyUpdate = [3]
wrongUpdate = [3, 3]
wrongUpdate2 = [1, 2, 3, 3, 3]